fdbbackup/FileDecoder.actor.cpp

```diff
@@ -215,7 +281,44 @@ std::string decode_hex_string(std::string line) {
 	return line.substr(0, i);
 }
 
-int parseDecodeCommandLine(DecodeParams* param, CSimpleOpt* args) {
+// Parses and returns a ";" separated HEX encoded strings. So the ";" in
+// the string should be escaped as "\;".
```

Comment:
Can this be configurable parameter instead? E.g. I think prefix per line can also be another common format we can get this input.

Reply:
I actually thought about doing that, but the `\r\n` and `\n` could easily make mistakes if we are not careful. So I avoided using `\n` as the separator.